core/src/Cuda/Kokkos_Cuda_GraphNodeKernel.hpp

```diff
@@ -128,15 +134,16 @@ struct get_graph_node_kernel_type<KernelType, Kokkos::ParallelReduceTag>
 // <editor-fold desc="get_cuda_graph_*() helper functions"> {{{1
 
 template <class KernelType>
-auto* allocate_driver_storage_for_kernel(KernelType const& kernel) {
+auto* allocate_driver_storage_for_kernel(const CudaSpace& mem,
+                                         KernelType const& kernel) {
```

Comment:
Is that `const` qualifier here the main reason why we ended up having to declare the graph node's `m_driver_storage` as `mutable`?

(not something that we should resolve here)